### ISSUE TITLE
MTB accusation protection

### ISSUE THREAD

**Issue by techtonik** [gratipay/inside.gratipay.com#857]
MTB is a "money transfer business". Has serious regulations - was one of the the factors in Gratipocalypse.

I think I now know enough to say that we can protect ourselves from accusation in being MTB :bike: by just explicitly saying that we are not.

**Gratipay is not money transfer business (MTB). We are a donation service for open source projects and payment support for open works.**.

Adding this into mission statement somewhere should be enough. Something like below, but shorter:

...Gratipay is an important technology to support the needs of people behind open source projects. Projects that together weave the invisible infrastructure that powers most of the web and gives creative freedom to new startups and initiatives. Our mission is to bring some love and humanity back into economy that doesn't accept openness and leaves people behind consumption of download stats. We provide the technology, and we can not guarantee that people will do with it...

In other words, putting explanation of what the service is into Terms of Usage should protect us. The only problem for lawyers is how to make this explanation human friendly.

**Comment by techtonik** [comment]
I am catching up! =)
